packages/debugger/lib/data/selectors/index.js

```diff
@@ -1041,8 +1041,16 @@ const data = createSelectorTree({
      * it only stores the ID
      */
     errorNode: createLeaf(
-      ["./errorNodeId", "/current/scopes/inlined"],
-      (id, scopes) => id !== undefined ? scopes[id].definition : null
+      ["./errorLocation", "/views/scopes/inlined"],
+      (errorLocation, scopes) => {
+        const sourceId = (errorLocation.source || {}).id;
+        const astId = (errorLocation.node || {}).id;
+        if (sourceId !== undefined && astId !== undefined) {
+          return scopes[sourceId][astId].definition;
+        } else {
+          return null;
+        }
```

Comment:
This could be refactored as a ternary expression, but certainly not a blocker.

Reply:
Hm, yeah, not going to bother I don't think.